src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,8 +1,18 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import mate.academy.spring.validation.Email;
+import mate.academy.spring.validation.Password;
+
+@Password
 public class UserRequestDto {
+    @Email
     private String email;
+    @NotNull
+    @Size(min = 4)
     private String password;
```

Comment:
```suggestion
    private String password;
```
move all checks to @Password